attachments_component/admin/config.xml

```diff
@@ -188,11 +188,12 @@
            label="ATTACH_CSS_STYLE_FOR_ATTACHMENTS_TABLES" size="40"
            description="ATTACH_CSS_STYLE_FOR_ATTACHMENTS_TABLES_DESCRIPTION">
     </field>
-    <field name="file_link_open_mode" type="radio" default="in_same_window" layout="joomla.form.field.radio.switcher"
+    <field name="file_link_open_mode" type="radio" default="in_same_window" class="btn-group btn-group-yesno"
            label="ATTACH_FILE_LINK_OPEN_MODE"
```

Comment:
I believe it is better presented as a list. Radio button groups are not very good at presenting a lot of text in smaller screens.
```suggestion
    <field name="file_link_open_mode" type="list" default="in_same_window"
```